plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -43,23 +44,30 @@ import {
 } from '../transformers';
 import { TechDocsBuildLogs } from './TechDocsBuildLogs';
 import { TechDocsNotFound } from './TechDocsNotFound';
+import { TechDocsSearch } from './TechDocsSearch';
 import { useReaderState } from './useReaderState';
 
 type Props = {
   entityId: EntityName;
   onReady?: () => void;
+  context?: string;
```

Comment:
I think some documentation for this prop could be good, or narrowing down the type if that makes sense

Reply:
+1 to documentation for this prop, or more usecases of the Reader component now when it can be used for different contexts, I think it can be done as part of this bigger documentation task though https://github.com/backstage/backstage/issues/3122#issuecomment-862177182

Reply:
Narrowing it down to
```suggestion
  context?: 'entitypage';
```
would be nice. Or would it even better if there was a value for the other case too instead of implicitly falling back to `undefined`?

Reply:
I'll add a specific string to determine that one special case